src/domain/collaboration/callout/callout.resolver.subscriptions.ts

```diff
@@ -92,4 +94,29 @@ export class CalloutResolverSubscriptions {
       SubscriptionType.CALLOUT_ASPECT_CREATED
     );
   }
+
+  @UseGuards(GraphqlGuard)
+  @TypedSubscription<CalloutMessageReceived, CalloutMessageReceivedArgs>(
```

Comment:
Should the type for the Payload not have Payload then in the name, similar to Args?